locales/da.json

```diff
@@ -0,0 +1,87 @@
+{
+    "Heisse Preise": "Spise med priser",
```

Comment:
Det her er nok det bedste forslag på et navn indtil videre, men jeg er ikke helt sikker på at jeg tør have navnet for tæt på et så kendt brand.